fintech-examples/fintech-db-schema/util/pg-create-schema.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
This script requires local pgsql installation. Why not to initialize postres using docker init?
i.e.:
docker run ... open_banking?TC_DAEMON=true&TC_INITSCRIPT=init.sql